docs/user/dql/functions.rst

```diff
@@ -1196,6 +1196,34 @@ Example::
     +---------------------+
 
 
+FROM_UNIXTIME
+-------------
+
+Description
+>>>>>>>>>>>
+
+Usage: Returns a representation of the argument given as a datetime or character string value. Perform reverse conversion for `UNIX_TIMESTAMP`_ function.
+If second argument is provided, it is used to format the result in the same way as the format string used for the `DATE_FORMAT`_ function.
+
+Argument type: DOUBLE, STRING
+
+Return type map:
+
+DOUBLE -> DATETIME
+
+DOUBLE, STRING -> STRING
+
+Example::
+
+    os> select FROM_UNIXTIME(1220249547), FROM_UNIXTIME(1220249547, '%T')
```

Comment:
Can these be split into two examples? It'd be easier to read that way.

Reply:
Thank you, updated in dfd2c1ea.